csharp-basic-topics/PerformSubstringSearchWithContainsMethod/Tests/StringSearchUnitTest.cs

```diff
@@ -0,0 +1,131 @@
+using PerformSubstringSearchWithContainsMethod;
+
+namespace Tests
+{
+    [TestClass]
+    public class StringSearchUnitTest
+    {
+        [TestMethod]
+        public void GivenSourceAndSubstring_WhenUsingStringContainsMethod_ThenShouldReturnContainmentResult()
+        {
+            var sourceString = "Code Maze";
+            var substringToSearch = "maze";
```

Comment:
Since each test is using the same arguments for sourceString and substringToSearch, rather than local variables in each method, we can make them class level constants.